ui/src/components/Layout/Search.vue

```diff
@@ -51,7 +52,6 @@ const results = computed(() => {
 .menubar-search {
   width: 350px !important;
   margin-right: 20px;
-  margin-top: 30px;
```

Comment:
Searchbar not lined up. seems like this line

![screenshot-localhost_3000-2022 04 25-15_06_18](https://user-images.githubusercontent.com/8680122/165157146-dfffea91-4700-407b-b785-c94d8008e378.png)